src/Workspaces/Core/Portable/Options/GlobalOptionService.cs

```diff
@@ -270,7 +270,7 @@ public SerializableOptionSet GetSerializableOptionsSnapshot(ImmutableHashSet<str
             var changedOptionsKeysSerializable = _changedOptionKeys
                 .Where(key => serializableOptions.Contains(key.Option) && (!key.Option.IsPerLanguage || languages.Contains(key.Language!)))
                 .ToImmutableHashSet();
-            return new SerializableOptionSet(languages, optionService, serializableOptions, serializableOptionValues, changedOptionsKeysSerializable);
+            return new SerializableOptionSet(languages, optionService, serializableOptionValues, changedOptionsKeysSerializable);
```

Comment:
this is the only location i can find where we really genuinely flow in this information.  And it is genuinely redundant with serializableOptionValues (the latter is computed from the former).

Reply:
@mavasani see comment here: https://github.com/dotnet/roslyn/pull/56655#discussion_r715770778